src/Uno.Extensions.Core.Generators/Common/GenContext/GenerationContext.cs

```diff
@@ -31,13 +31,19 @@ public static bool IsDisabled(this GeneratorExecutionContext context, string dis
 					x.parameter,
 					type: x.attribute.Type,
 					isOptional: x.attribute.IsOptional || x.parameter.GetCustomAttributesData().Any(attr => attr.AttributeType.FullName.Equals("System.Runtime.CompilerServices.NullableAttribute")),
-					symbol: compilation!.GetTypeByMetadataName(x.attribute.Type)
+					symbol: compilation
+						.GetTypesByMetadataName(x.attribute.Type)
+						.OrderBy(t =>
+							SymbolEqualityComparer.Default.Equals(t.ContainingAssembly, compilation.Assembly) ? 0
```

Comment:
Switch expression instead?

Reply:
Would be hackish since conditions are only method based, but yes why not. Will update, but in another commit if the build finally pass.